archiver_test.go

```diff
@@ -203,10 +203,10 @@ func (s *ArchiverSuite) TestFixtures() {
 			require := require.New(t)
 			var hash model.SHA1
 
-			or, err := ct.OldRepository()
+			or, _ := ct.OldRepository()
```

Comment:
should we check the error instead of skip it?

Reply:
Done